core/modules/widgets/checkbox.js

```diff
@@ -208,16 +208,20 @@ CheckboxWidget.prototype.handleChangeEvent = function(event) {
 	if(this.checkboxListField || this.checkboxListIndex) {
 		var fieldContents, listContents, oldPos, newPos;
 		if(this.checkboxListField) {
-			fieldContents = tiddler ? tiddler.fields[this.checkboxListField] : undefined;
+			fieldContents = (tiddler ? tiddler.fields[this.checkboxListField] : undefined) || [];
 		} else {
 			fieldContents = this.wiki.extractTiddlerDataItem(this.checkboxTitle,this.checkboxListIndex);
 		}
 		if($tw.utils.isArray(fieldContents)) {
 			// Make a copy so we can modify it without changing original that's refrenced elsewhere
 			listContents = fieldContents.slice(0);
-		} else {
-			listContents = $tw.utils.parseStringArray(fieldContents) || [];
```

Comment:
@rmunn @Jermolene This change causes this problem: https://talk.tiddlywiki.org/t/checkboxwidget-listindex-mode-no-longer-working-in-5-3-0/7460